src/snowcli/cli/nativeapp/init.py

```diff
@@ -79,11 +81,11 @@ def render_snowflake_yml(parent_to_snowflake_yml: Path):
 
     try:
         generic_render_template(
-            template_path=parent_to_snowflake_yml.joinpath(snowflake_yml_jinja),
+            template_path=parent_to_snowflake_yml / snowflake_yml_jinja,
```

Comment:
All changes that remove `joinpath` and instead use `/` were suggested as part of a previous PR on NADE's fork that was closed in favor of this PR.